AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -75,6 +75,9 @@
     private final Map<String, RemoteParticipant> remoteParticipantsMap;
     private final List<RemoteParticipant> displayedRemoteParticipants;
     private final MutableLiveData<List<RemoteParticipant>> displayedParticipantsLiveData;
+
+    private final MutableLiveData<RemoteParticipantUpdate> remoteParticipantUpdate;
```

Comment:
not sure if spaces around this line are required

Reply:
Removed